### PR TITLE
Add support for Razer Blade 15 Early 2020 base

### LINKED ISSUE #1150: Support for Blade 15 Base Model (2020)
Doesn't look like the 2020 blade is supported? I wouldn't think it should be too different from the 2019 model but what do I know. Will try and get pcaps tomorrow.

### Device Info

* Name: Blade 15 Base Model (2020)
* Product Number: RZ09-03287E72
* Model Number: RZ09-0328

### Device Information
```
Please post the output of
    lsusb -d 1532: -v
```
```
Bus 001 Device 003: ID 1532:0255 Razer USA, Ltd Razer Blade
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0
  bDeviceSubClass         0
  bDeviceProtocol         0
  bMaxPacketSize0        64
  idVendor           0x1532 Razer USA, Ltd
  idProduct          0x0255
  bcdDevice            2.00
  iManufacturer           1 Razer
  iProduct                2 Razer Blade
  iSerial                 0
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength       0x0054
    bNumInterfaces          3
    bConfigurationValue     1
    iConfiguration          0
    bmAttributes         0xa0
      (Bus Powered)
      Remote Wakeup
    MaxPower              500mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              0
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      61
         Report Descriptors:
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        1
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0
      bInterfaceProtocol      1 Keyboard
      iInterface              0
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength     159
         Report Descriptors:
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x82  EP 2 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0010  1x 16 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        2
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0
      bInterfaceProtocol      2 Mouse
      iInterface              0
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      94
         Report Descriptors:
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x83  EP 3 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1
can't get device qualifier: Resource temporarily unavailable
can't get debug descriptor: Resource temporarily unavailable
Device Status:     0x0000
  (Bus Powered)```
```

```
Please post the output of
    ls -lah /dev/input/by-id/
```
```
total 0
drwxr-xr-x 2 root root 160 May 15 21:19 .
drwxr-xr-x 4 root root 660 May 15 21:19 ..
lrwxrwxrwx 1 root root   9 May 15 21:19 usb-Azurewave_USB_Camera_0x0001-event-if00 -> ../event3
lrwxrwxrwx 1 root root   9 May 15 21:19 usb-Razer_Razer_Blade-event-if01 -> ../event9
lrwxrwxrwx 1 root root   9 May 15 21:19 usb-Razer_Razer_Blade-event-kbd -> ../event5
lrwxrwxrwx 1 root root   9 May 15 21:19 usb-Razer_Razer_Blade-if01-event-kbd -> ../event6
lrwxrwxrwx 1 root root  10 May 15 21:19 usb-Razer_Razer_Blade-if02-event-mouse -> ../event10
lrwxrwxrwx 1 root root   9 May 15 21:19 usb-Razer_Razer_Blade-if02-mouse -> ../mouse2
```

### Packet Captures

<Please see here as we probably need packet captures>
<https://github.com/openrazer/openrazer/wiki/Reverse-Engineering-USB-Protocol>

### Synapse Screenshots
<Screenshots of all available options as seen on Razer Synapse>

### PR DESCRIPTION
**Pull request by vvvrrooomm**
Fixes #1150

### CODE REVIEW

**Review by z3ntu — APPROVED** [review]
LGTM, I've asked in #1150  for some testers so let's wait a bit before merging.

File: driver/razerkbd_driver.c
Comment (zhassanbey) [inline driver/razerkbd_driver.c]: Shouldn't that be `device_type = "Razer Blade 15 Base (Early 2020)\n"` instead?

**Review by zhassanbey — CHANGES_REQUESTED** [review]
Fix device_type in ` case USB_DEVICE_ID_RAZER_BLADE_EARLY_2020_BASE:` line 415.

Could you also have a look at the issue I posted earlier after I tested this change, https://github.com/openrazer/openrazer/issues/1150#issuecomment-673903563

Thanks.

**Review by zhassanbey — APPROVED** [review]

**Comment by FranciscodeMaussion** [comment]
Sorry if is a bad place to post a question, but Idk how to test it, may i have to download and make it my self? Will this be merged to master soon? and to arch repositories?
EDIT: Didn't notice the wiki, all the info was there. Thanks, waiting for release. But for now, works like a charm.

**Comment by z3ntu** [comment]
Thanks!

**Pull request merged**
